extensions/resteasy-jsonb/deployment/src/main/java/io/quarkus/resteasy/jsonb/deployment/JsonbConfig.java

```diff
@@ -0,0 +1,61 @@
+package io.quarkus.resteasy.jsonb.deployment;
+
+import static io.quarkus.runtime.annotations.ConfigPhase.BUILD_TIME;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.Set;
+
+import javax.json.bind.config.PropertyOrderStrategy;
+
+import io.quarkus.runtime.annotations.ConfigItem;
+import io.quarkus.runtime.annotations.ConfigRoot;
+
+@ConfigRoot(phase = BUILD_TIME)
+public class JsonbConfig {
+
+    static final Set<String> ALLOWED_PROPERTY_ORDER_VALUES = Collections.unmodifiableSet(
+            new HashSet<>(Arrays.asList(PropertyOrderStrategy.LEXICOGRAPHICAL, PropertyOrderStrategy.ANY,
+                    PropertyOrderStrategy.REVERSE)));
+
+    /**
+     * default locale to use
+     */
+    @ConfigItem(defaultValue = "")
+    String locale;
+
+    /**
+     * default date format to use
+     */
+    @ConfigItem(defaultValue = "")
+    String dateFormat;
+
+    /**
+     * defines whether or not null values are serialized
+     */
+    @ConfigItem(defaultValue = "false")
+    boolean serializeNullValues;
+
+    /**
+     * defines the order in which the properties appear in the json output
+     */
+    @ConfigItem(defaultValue = PropertyOrderStrategy.LEXICOGRAPHICAL)
+    String propertyOrderStrategy;
+
+    /**
+     * encoding to use when deserializing data
+     */
+    @ConfigItem(defaultValue = "")
+    String encoding;
+
+    /**
+     * specified whether unknown properties will cause deserialization to fail
+     */
+    @ConfigItem(defaultValue = "true")
+    boolean failOnUnknownProperties;
```

Comment:
Today Yasson don't fail on unknown properties so `false` should be the default. IMHO it's a good default for microservices because it allow a service to "remove" an attribute without changing the client.

Reply:
According to [this](https://github.com/eclipse-ee4j/yasson/blob/master/src/main/java/org/eclipse/yasson/YassonProperties.java#L23), the default value is true.

I agree with you that `false` is probably a better default and I  am inclined to change it. Let's see what others think as well

Reply:
It's strange because I tested it before commenting and I can send unknown properties to a restendpoint and yasson don't generate any error.

Reply:
In any case I agree that `false` should be the default,  but I would like to see what others think about changing the default value of well known library